core/src/components/toggle/toggle.scss

```diff
@@ -134,47 +134,53 @@ input {
   align-items: center;
 }
 
-// Toggle Justify
-// --------------------------------------------------
+// Toggle Bottom Content
+// ----------------------------------------------------------------
+
+.toggle-bottom {
+  @include padding(4px, null, null, null);
```

Comment:
this should be on the native partial and on Ionic theme use the proper space token.